content/Arduino Cloud/Arduino IoT Cloud/What-devices-can-be-used-with-Arduino-IoT-Cloud.md

```diff
@@ -21,7 +21,19 @@ The following Arduino boards can be used with IoT Cloud:
 * [Nano 33 IoT](https://docs.arduino.cc/hardware/nano-33-iot)
 * [Portenta H7](https://docs.arduino.cc/hardware/portenta-h7)
 * [Portenta H7 Lite Connected](https://docs.arduino.cc/hardware/portenta-h7-lite-connected)
+* [Portenta X8](https://docs.arduino.cc/hardware/portenta-x8)
 * [Nicla Vision](https://docs.arduino.cc/hardware/nicla-vision)
+* [Arduino Opta](https://docs.arduino.cc/hardware/opta)
```

Comment:
Opta is not released yet. I would include this now if we are certain that Opta will work in the IoT Cloud